### DISCUSSION TITLE
I'm so confused. Why does "SH4 Clock" setting say it may "trigger unexpected nuclear reactions."?

### DISCUSSION THREAD

**Discussion by andygun111** [flyinghead/flycast#1441] [Q&A]
Why does "SH4 Clock" setting say it may "trigger unexpected nuclear reactions."?
I'm fairly sure it's a joke, but I don't get it.

**Comment by cinnamonmatexfce** [comment]
Because surely it'll cause reactor's kernel meltdown

**Comment by TheCoolPup** [comment]
Cause secretly the sh4 clock is secretly a pipe bomb :)
